services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -502,45 +502,89 @@ struct DimensionSizes {
     pub lon: usize,
 }
 
-#[derive(Debug)]
-struct GroupPath<'g> {
-    pub prefix: Vec<GroupPathSegment>,
-    pub current: MdGroup<'g>,
+#[derive(Debug, Clone, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct NetCdfOverview {
+    pub file_name: String,
+    pub title: String,
+    pub spatial_reference: SpatialReferenceOption,
```

Comment:
why Option, do we support non-georeferenced EBVs?

Reply:
[9022b1f](https://github.com/geo-engine/geoengine/pull/467/commits/9022b1fdf49c2c486bd2dd3c05cc62061b519146)